lib/models/user/models.dart

```diff
@@ -1,12 +1,67 @@
+// To parse this JSON data, do
+//
+//     final userProfile = userProfileFromJson(jsonString);
+
 import 'package:freezed_annotation/freezed_annotation.dart';
+import 'dart:convert';
 
 part 'models.freezed.dart';
 part 'models.g.dart';
 
+UserProfile userProfileFromJson(String str) =>
+    UserProfile.fromJson(json.decode(str));
+
+String userProfileToJson(UserProfile data) => json.encode(data.toJson());
```

Comment:
Why do you need those methods?

Reply:
Oh, I've just missed that from auto-gen https://app.quicktype.io/
my bad